src/style/disease-table.module.css

```diff
@@ -44,6 +44,11 @@
     display: none; /* remove dividers between column headers */
 }
 
+.container :global(.ant-table-row) .hovered-row:global(.ant-table-cell) {
```

Comment:
do you need the `:global(.ant-table-cell)` here?

Reply:
I need it for specificity, working with some of your changes if I use this:
```
.container .data-complete .hovered-row:global(.ant-table-cell) 
```
it works, and if I take out the `:global(.ant-table-cell)` and add `!important` it also works, so I think
`.container .data-complete .hovered-row` targets the cells but isn't specific enough to override some other rule.